components/sidebar/sidebar_header/dropdown/index.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I understand we want to migrate this component from the legacy folder to here. But, I would think it's better to do that in another ticket. We could create other tickets to do some of that cleaning up. We want to reduce the scope of this PR to only the ticket at hand in order to avoid any regressions we didn't see coming.